index.html

```diff
@@ -51,6 +51,16 @@
   <body>
     <h1 id="header">Basics: Beat That! 🚀</h1>
     <div id="container">
+      <p>Welcome to Beat That!</p>
+      <p><br /></p>
+      <p>You will first roll two dice.</p>
+      <p>
+        Next, create a two-digit number by selecting the order of your dice
+        rolls. Option 1 indicates first roll digit followed by the second roll
+        digit. Option 2 is the opposite of option 1.
+      </p>
+      <p>The player with the highest number wins!</p>
+      <p><br /></p>
```

Comment:
good attempt at editing HTML and a little bit of CSS (even if it is just background color lol)

hope to see more of this in blackjack!